platform/linux-generic/arch/default/odp_cpu_idling.h

```diff
@@ -8,6 +8,7 @@
 
 #ifndef ODP_DEFAULT_CPU_IDLING_H_
 #define ODP_DEFAULT_CPU_IDLING_H_
+#include <odp/api/cpu.h>
```

Comment:
This is likely not needed as doze() has been removed already. Please rebase your PR to latest master and remove this change.